cmd/device-util/main.go

```diff
@@ -3,105 +3,40 @@
 package main
 
 import (
-	"errors"
 	"fmt"
+	"log"
 	"os"
 	"strings"
 
-	"github.com/Microsoft/hcsshim/internal/windevice"
-	"github.com/Microsoft/hcsshim/internal/winobjdir"
-	"github.com/urfave/cli"
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	cli "github.com/urfave/cli/v2"
 )
 
-const usage = `device-util is a command line tool for querying devices present on Windows`
+const desc = `A stand-alone tool that replicates a limited subset of sc.exe and other tools.
+It can be used within WCOW uVMs and containers without requiring additional DLLs.`
 
 func main() {
-	app := cli.NewApp()
-	app.Name = "device-util"
-	app.Commands = []cli.Command{
-		queryChildrenCommand,
-		readObjDirCommand,
+	app := &cli.App{
+		Name:        "device-util",
+		Usage:       "tool for managing devices, drivers, and services on Windows",
+		Description: desc,
+		Commands: []*cli.Command{
+			driverCommand,
+			serviceCommand,
+			queryChildrenCommand,
+			readObjDirCommand,
+		},
 	}
-	app.Usage = usage
 
 	if err := app.Run(os.Args); err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
+		log.New(os.Stderr, "", 0).Fatal(err)
```

Comment:
Why not just `log.Fatal(err)`? I assume the overhead to print date/time is very low.

Reply:
it looked cluttered, and if that output was going to get captured from within the guest and logged or surfaced up, I didnt want the timestamp in the error message

Reply:
Honestly in that case it seems easier to just print to stderr and exit directly like we did before. Feel like using `log` here is just complicating things.